griptape-aws-bill-pdf-to-csv/README.md

```diff
@@ -31,20 +25,31 @@ This sample converts an AWS bill PDF into a CSV. This is most useful for custome
 You can run this Structure locally by providing the following inputs:
 
 ```
-python structure.py -s <source_s3_uri> -d <destination_s3_uri>
+python structure.py -b <bucket_id> -d <workdir> -p <pdf_asset_file_name> -c <csv_asset_file_name>
+```
+
+You may also run the `upload.py` and `download.py` scripts locally to upload and download files directly:
+```
+python upload.py -b <bucket_id> -p <local_file_path> -d <workdir> -n <asset_file_name>
+
+python download.py -b <bucket_id> -p <local_file_path> -d <workdir> -n <asset_file_name>
 ```
 
 ### Griptape Cloud
 
 You can create a run via the API or the UI. When creating runs in the UI, you will need to specify parameters on their own lines.
 
 ```
--s
-<source_s3_uri>
+-b
+<bucket_id>
 -d
-<destination_s3_uri>
+<workdir>
+-p
+<pdf_asset_file_name>
```

Comment:
is this relative to the bucket_id then?

Reply:
Yes, all asset names are relative to the bucket they are in

Reply:
Can we make that more clear either in this README or by providing an example? Because that was not clear to me when reading this.

Reply:
I'll try to make clearer